{{cookiecutter.project_slug}}/Makefile

```diff
@@ -1,60 +1,57 @@
-CODE_PATH?={{cookiecutter.code_directory}}
+##### PATHS #####
+
 DATA_PATH?=data
+CODE_PATH?={{cookiecutter.code_directory}}
 NOTEBOOKS_PATH?=notebooks
-REQUIREMENTS_PIP?=requirements.txt
-REQUIREMENTS_APT?=apt.txt
 RESULTS_PATH?=results
-PROJECT_PYTHON_FILES=setup.cfg
+
+PROJECT_FILES=requirements.txt apt.txt setup.cfg
 
 PROJECT_PATH_STORAGE?=storage:{{cookiecutter.project_slug}}
-CODE_PATH_STORAGE?=$(PROJECT_PATH_STORAGE)/$(CODE_PATH)
-DATA_PATH_STORAGE?=$(PROJECT_PATH_STORAGE)/$(DATA_PATH)
-NOTEBOOKS_PATH_STORAGE?=$(PROJECT_PATH_STORAGE)/$(NOTEBOOKS_PATH)
-REQUIREMENTS_PIP_STORAGE?=$(PROJECT_PATH_STORAGE)/$(REQUIREMENTS_PIP)
-REQUIREMENTS_APT_STORAGE?=$(PROJECT_PATH_STORAGE)/$(REQUIREMENTS_APT)
-RESULTS_PATH_STORAGE?=$(PROJECT_PATH_STORAGE)/$(RESULTS_PATH)
 
 PROJECT_PATH_ENV?=/{{cookiecutter.project_slug}}
-CODE_PATH_ENV?=$(PROJECT_PATH_ENV)/$(CODE_PATH)
-DATA_PATH_ENV?=$(PROJECT_PATH_ENV)/$(DATA_PATH)
-NOTEBOOKS_PATH_ENV?=$(PROJECT_PATH_ENV)/$(NOTEBOOKS_PATH)
-REQUIREMENTS_PIP_ENV?=$(PROJECT_PATH_ENV)/$(REQUIREMENTS_PIP)
-REQUIREMENTS_APT_ENV?=$(PROJECT_PATH_ENV)/$(REQUIREMENTS_APT)
-RESULTS_PATH_ENV?=$(PROJECT_PATH_ENV)/$(RESULTS_PATH)
 
-NEURO?=neuro
-NEURO_CP=$(NEURO) cp --recursive --update --no-target-directory
+##### JOB NAMES #####
 
 PROJECT_POSTFIX?={{cookiecutter.project_slug.replace('_', '-')}}
+
 SETUP_NAME?=setup-$(PROJECT_POSTFIX)
 TRAINING_NAME?=training-$(PROJECT_POSTFIX)
 JUPYTER_NAME?=jupyter-$(PROJECT_POSTFIX)
 TENSORBOARD_NAME?=tensorboard-$(PROJECT_POSTFIX)
 FILEBROWSER_NAME?=filebrowser-$(PROJECT_POSTFIX)
 
+##### ENVIRONMENTS #####
+
 BASE_ENV_NAME?=neuromation/base
 CUSTOM_ENV_NAME?=image:neuromation-$(PROJECT_POSTFIX)
-TRAINING_MACHINE_TYPE?=gpu-small
 
-# Set it to True (verbatim) to disable HTTP authentication for your jobs
-DISABLE_HTTP_AUTH:=
-ifeq ($(DISABLE_HTTP_AUTH), True)
-	HTTP_AUTH:=--no-http-auth
-endif
+##### COMMANDS #####
 
-APT_COMMAND?=apt-get -qq
-PIP_INSTALL_COMMAND?=pip install --progress-bar=off
-# example:
-# TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_PATH)/train.py --data $(DATA_PATH_ENV)'"
+APT?=apt-get -qq
+PIP?=pip install --progress-bar=off
+NEURO?=neuro
+
+##### VARIABLES YOU MAY WANT TO MODIFY #####
+
+# Location of your dataset on the platform storage. Example:
+# DATA_PATH_STORAGE?=storage:datasets/cifar10
```

Comment:
let's change the value in this example to a more realistic value: `storage:my-cool-project/data/cifar10`

Reply:
Disagree. The reason for changing this variable is having your data unassociated with your project. Thus it makes sense to show that the data may be somewhere else on your storage rather than inside your project structure.